### ISSUE TITLE
Pure White LEDs not working

### ISSUE THREAD

**Issue by reg-is** [steve228uk/homebridge-magichome#12]
I can only control the colour LEDs but not the pure white LEDs from my RGBW-Strip.
I found a [pull request](https://github.com/beville/flux_led/pull/21) for belleville's flux_led.py that works great for me. Is there a way to add it in this homebridge extension?

**Comment by djdom111** [comment]
Hi I have the same problem as you do/did. Have you found a solution? Did you maybe modify the file? Is any other solution to make the RGBW strip work with the home bridge-magichome?

**Comment by powneill** [comment]
Just chiming in to say that the Ortner's fix for whites was exactly what I needed.  I now have a WW-CW (CCT) combined strip that is working perfectly and fully adjustable in HomeKit, using:
"setup":"RGBW",
"purewhite":false,
"onlywhite":true,
"combinedwhite":false

A slight word of caution:  For my project I am using [these lights](https://www.amazon.com/gp/product/B00JPCFRYS/) and [this controller](https://www.amazon.com/gp/product/B01DY56N8U/). When I initially tested this fork I erroneously had both onlywhite and combinedwhite set to true...  When I tried it out within HomeKit, I heard my power supply fan kick in to high gear (which should never have occurred  - I am using less than 20% of its capacity currently),  I turned around,  and the LED strip was just slightly less bright than the surface of the sun. When I looked at the app used to natively control the controller (MagicHome Pro), the brightness setting said 200% (!).  Somehow I got to a point where the controller was attempting to overdrive the strip, big-time.  I got it shut down, and only time will tell if it did some damage to the strip, but it was definitely sight to behold while it was happening...

TL;DR version - this controller has the capability of going to 11 if you are not careful.
